extensions/google-drive/src/utils.ts

```diff
@@ -161,6 +164,33 @@ ${new Date(file.createdAt).toLocaleString()}
 ${new Date(file.updatedAt).toLocaleString()}`;
 };
 
+export const fileStream = ({ stats = false }: { stats?: boolean } = {}) => {
```

Comment:
Can we name this explicitly to something like `drivePathsStream()` since it is not a generic stream/transform?

Reply:
Resolved in [397e485](https://github.com/raycast/extensions/pull/1613/commits/397e48545d88d8c9a479a66a88f434e896478825)